docs/specs/source_python_math.tex

```diff
@@ -9,7 +9,9 @@ \subsection*{MATH Library}
 \texttt{Math} library, see\\
 \href{https://www.ecma-international.org/ecma-262/9.0/index.html\#sec-math-object}{\color{DarkBlue}ECMAScript Specification, Section 20.2}. Examples:
```

Comment:
Refer to Python documentation here, not to ECMAScript. Make sure that all Python math functions and only the Python math functions are available as math_...